{"name": "测试用例描述：错误的验证码登录失败", "status": "passed", "description": "测试描述：错误的验证码登录失败", "steps": [{"name": "输入正确账号", "status": "passed", "attachments": [{"name": "正确账号", "source": "b5185dba-114a-4cf6-a07f-d1f1a17f6a79-attachment.attach"}], "start": 1651817971422, "stop": 1651817971422}, {"name": "输入正确密码", "status": "passed", "attachments": [{"name": "正确密码", "source": "14c034ac-4ae4-4b28-8327-9ac45e563473-attachment.attach"}], "start": 1651817971481, "stop": 1651817971481}, {"name": "输入错误验证码", "status": "passed", "attachments": [{"name": "验证码为空", "source": "13f5ab1a-6ae7-4e05-9b1b-014297ba8826-attachment.attach"}], "start": 1651817971534, "stop": 1651817971534}], "attachments": [{"name": "截图", "source": "1719ea59-bc0e-4206-84ce-56132aedc518-attachment.png", "type": "image/png"}], "start": 1651817971422, "stop": 1651817974121, "uuid": "d982693f-0b66-4724-9ff2-1da2e0497ea1", "historyId": "99cc6b199ab7eac5979ecaecd144eda1", "testCaseId": "44097febf60fbfed56a361cecfc05a9e", "fullName": "test_two.Test#test_07", "labels": [{"name": "story", "value": "登录"}, {"name": "epic", "value": "xxx测试"}, {"name": "severity", "value": "critical"}, {"name": "feature", "value": "登录模块"}, {"name": "suite", "value": "test_two"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "1256-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_two"}]}